src/main/resources/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -200,7 +200,31 @@
       "mavenCoordinates": {
          "groupId": "com.google.cloud",
          "artifactId": "google-cloud-errorreporting",
-         "version": "0.29.0-beta"
+         "version": "0.30.0-beta"
+       }
+      }
+    ]
+  },
+  {
+    "name": "Google Cloud Firestore",
+    "id": "cloudfirestore",
+    "serviceName": "firestore.googleapis.com",
```

Comment:
Did you confirm that no roles are associated with this service?

Reply:
(im trying to check this myself too)

Reply:
I can't find any documentation of any roles anywhere. Where should I look?

Reply:
I didn't find any. You can get a list of all available roles from the Iam api:

`iam.roles().list()`

approving

Reply:
fyi its also available via gcloud:

https://cloud.google.com/sdk/gcloud/reference/iam/list-grantable-roles